docs/installation/osx.md

```diff
@@ -86,11 +86,15 @@ Follow the step-by-step instructions below:
 
     Do not forget to use `bundle exec` in front of every rake/rails command.
 
-12. Create initial root user
+12. Create initial root user. (Flag is required for differentiating the environment because this affects the database (prod or dev) in which the user is created.)
```

Comment:
Nit: just rephrase this a little bit (e.g. "pass the `-d` flag for developmental deployments). Not a blocker though

Reply:
Done, could you approve this again? Thanks!